python/ql/src/experimental/semmle/python/security/InsecureRandomness.qll

```diff
@@ -21,17 +21,14 @@ module InsecureRandomness {
    * A taint-tracking configuration for reasoning about random values that are
    * not cryptographically secure.
    */
-  class Configuration extends TaintTracking::Configuration {
-    Configuration() { this = "InsecureRandomness" }
+  private module Config implements DataFlow::ConfigSig {
+    predicate isSource(DataFlow::Node source) { source instanceof Source }
 
-    override predicate isSource(DataFlow::Node source) { source instanceof Source }
+    predicate isSink(DataFlow::Node sink) { sink instanceof Sink }
 
-    override predicate isSink(DataFlow::Node sink) { sink instanceof Sink }
-
-    override predicate isSanitizer(DataFlow::Node node) { node instanceof Sanitizer }
-
-    deprecated override predicate isSanitizerGuard(DataFlow::BarrierGuard guard) {
-      guard instanceof SanitizerGuard
-    }
+    predicate isBarrier(DataFlow::Node node) { node instanceof Sanitizer }
   }
+
+  /** Global taint-tracking for detecting "random values that are not cryptographically secure" vulnerabilities. */
+  module Flow = TaintTracking::Global<Config>;
```

Comment:
Is there a good reason why this uses the alternative naming convention (option 1)?

Reply:
I tried to keep changes to a minimal. Since it was already inside a properly named module, I let things stay that way.

So no better reason than I tried to get things done :sweat_smile: